docs/language-server/protocol-language-server.md

```diff
@@ -367,7 +367,7 @@ interface ExpressionUpdate {
   /** The id of updated expression. */
   expressionId: ExpressionId;
   /** The updated type of the expression. */
-  type?: string;
+  type: string[];
```

Comment:
I feel incompatible changes should be versioned and documented. We should:
- increase the version of the protocol
- describe what change has happened and why it is incompatible


Sure, I know we have no version of the protocol. However I asked for it a year ago. Moreover recently there were complains about incompatibilities between cloud/IDE and the conclusion was: we should at least detect the incompatibilities by versioning our protocols.